draft-ietf-lamps-kyber-certificates.md

```diff
@@ -640,111 +639,110 @@ key in the previous section.
+         :     4A C3 CD 29 C7 F2 09 24 10 C5 07 83 55 B1 38 FB
+         :     53 A6 B9 AE 6E 0B 9C 08 24 3E 7B AA 45 C4 73 76
+         :     EB 8C 7F 13 D4 CF 51 AA 73 6F A3 15 40 C9 24 1F
+         :     37 0D A5 44 BF 9F 9C 28 D9 A5 7E 2F 2A 7C A9 5A
+         :     4E 4B 46 6E 64 1A B3 BC C7 6A DF 11 39 D5 67 A6
+         :     F1 2B 52 F3 A6 5E 7E C0 AA E2 6B CA A8 C5 58 33
+         :     B0 4E 59 99 8E BC 9A 19 30 FB B6 D2 23 3C 53 D2
+         :     C1 F8 B9 51 8E 3C 2D E7 3A 19 DE E6 B3 80 A5 B3
+         :     29 71 CF 64 E1 29 FD 6C 1F A6 E7 5D 4A 23 45 01
+         :     E9 66 DD 3A 54 0A F5 C8 F4 F3 4A 6B 4A 25 3E E2
+         :     84 92 56 6D 5E 67 C6 F5 58 55 FC B0 50 6F B0 6C
+         :     15 67 44 D9 A0 3A 31 A2 6F A9 4C AD 14 F1 57 B7
+         :     F3 03 D0 7A 69 C7 73 76 8F CB 4D 07 9C 09 05 97
+         :     03 A0 C3 A9 4D E4 B9 9E A3 A2 F1 65 83 D0 F9 17
+         :     0A 39 50 DB 07 B4 F0 BC 30 80 29 27 F9 F7 96 1B
+         :     62 59 89 26 36 A9 50 2A 27 05 30 36 37 79 9D D3
+         :     44 DA 45 1C 1C F7 BF 67 84 0C EB 30 79 AB 8C 6B
+         :     8C 19 27 F6 40 53 C6 12 45 0C 45 C9 E6 03 BC 16
+         :     66 6E 59 6B 34 71 E1 03 B6 F1 54 47 42 4D 17 02
+         :     20 48 11 1F FB D3 7E 1C 67 0F 64 F1 4B 8A 7B 32
+         :     B9 4C 1A 49 B4 5D D2 FC 38 CD 52 89 D9 10 AD 63
+         :     60 2C F5 E1 30 42 C6 4A C6 79 7B 89 FB 55 1A D0
+         :     8E 05 A9 2D 20 0C CC B7 E7 12 EF 23 C9 31 2C B3
+         :     50 F0 29 AB 53 7E 28 73 47 FD 30 75 AC 10 90 6A
+         :     78 3F 1C 6C 07 CC B8 8F 41 22 8C 4B E1 C6 40 F7
+         :     90 B5 C3 A5 D5 D3 CA 79 24 95 D7 4B C4 61 56 26
+         :     58 C0 7A C6 00 27 6B 92 4A B5 BC 9B E1 F0 49 4C
+         :     B7 6F 82 F4 60 A7 48 09 72 66 33 81 E1 69 99 60
+         :     61 D7 99 85 9E C5 4D 4F 5C A5 C4 11 C0 1D B1 59
+         :     7B 16 59 77 66 9D E1 3A 92 8A 34 AF BA C2 58 FE
+         :     A8 C4 76 42 39 C9 42 1D C3 11 9B F5 B4 76 99 20
+         :     69 78 32 7B 1C 53 45 EF 74 6A 79 83 84 1F 05 6E
+         :     25 34 10 0A B2 4D 4E 9A BB D0 B1 7C 6A 95 BD 4C
+         :     3C 0E 40 F6 9E 16 12 AC EE B2 8B 99 08 6C 95 11
+         :     6E 72 04 27 38 93 39 0B F4 6B 89 9B 36 28 6B 0E
+         :     BF 19 47 BB 98 84 F7 32 CA 27 DA 82 B1 9B 5D C0
+         :     CC 7F 88 85 71 49 10 88 8B 23 10 C4 F9 31 9D 41
+         :     0B 34 E6 43 3B 90 03 E2 17 6B B9 95 25 74 56 10
+         :     6E 89 52 16 3B 8B A5 92 53 0C C5 AA 0A EB 43 AD
+         :     39 8F E9 E9 7B AA 52 3D 7A 44 31 67 7C 3D 3A F0
+         :     71 9E 47 5D B8 5C A9 5A F5 08 9B EA BE B0 5B 2F
+         :     AA B4 89 6B A6 0F 81 C8 84 72 A5 7B 46 A8 28 82
+         :     6A 0C DF B4 46 F8 18 91 82 D2 BF 5E AC 4E C1 CC
+         :     5D EA F5 99 C8 A1 3E 48 23 54 06 D1 7F FD DC 83
+         :     44 B6 C6 69 84 A8 68 AA 92 FA 02 22 7A 08 69 50
+         :     EB 0C 87 01 ED 58 DC 62 87 76 B9 83 88 2E 11 75
+         :   }
 ~~~
 
```

Comment:
<aside markdown="block">
NOTE: The first byte of the BIT STRING's content, the padding byte, has been stripped from the pretty print above.
</aside>